integration/setup/common.go

```diff
@@ -74,6 +74,28 @@ func SkipForTigrisWithReason(tb testing.TB, reason string) {
 	}
 }
 
+// SkipForPostgresWithReason skips the current test for Postgres (pg) handler.
+//
+// Ideally, this function should not be used. It is allowed to use it in Tigris-specific tests only.
+func SkipForPostgresWithReason(tb testing.TB, reason string) {
+	tb.Helper()
+
+	if *handlerF == "pg" {
+		tb.Skipf("Skipping for Postgres", reason)
+	}
+}
+
+// SkipForMongoWithReason skips the current test for Mongo (mongo) handler.
+//
+// Ideally, this function should not be used. It is allowed to use it in Tigris-specific tests only.
```

Comment:
Please update that comment

Reply:
@AlekSi, what part of the comment do you disagree with? That it is Tigris-specific?

Reply:
> for Mongo (mongo) handler.

It should be "MongoDB (mongodb)"

Reply:
I've just fixed that one! (But it's two lines above).